pg-core/src/java/org/pg/Connection.java

```diff
@@ -49,6 +51,7 @@ public final class Connection implements AutoCloseable {
     private String unixSocketPath;
     private InputStream inStream;
     private OutputStream outStream;
+    private AsynchronousSocketChannel sockCh;
```

Comment:
I might be wrong, but it looks like the `sockCh` field is not needed here. We store both input and output streams, and a raw socket object for possible SSL upgrade. So I think this reference can be skipped.

Reply:
You're right. It can be removed